dask/array/tests/test_fft.py

```diff
@@ -137,6 +137,7 @@ def test_wrap_ffts(modname, funcname, dtype):
 
     wfunc = fft_wrap(func)
     assert wfunc(darrc).dtype == func(nparrc).dtype
+    assert wfunc(darrc).shape == func(nparrc).shape
```

Comment:
PR ( https://github.com/dask/dask/pull/2101 ) should fix `assert_eq` to test the shapes before computation, but it looks like it will expose some existing issues. Not sure if I will have time to fix them all though.